src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotMethodData.java

```diff
@@ -571,9 +566,8 @@ public int getExecutionCount(HotSpotMethodData data, int position) {
             return -1;
         }
 
-        @Override
         protected long getTypesNotRecordedExecutionCount(HotSpotMethodData data, int position) {
```

Comment:
I think you can inline this method into the callers as it only existed to distinguish the count and nonprofiled_count